src/book.py

```diff
@@ -87,18 +106,26 @@ def get_content(self):
 
     def save_book(self):
         # Save the book in md format
-        with open(f'book.md', 'w') as file:
+        # Corrected saving the book with the specified time
+        desired_time = datetime.now(timezone(timedelta(hours=-5)))  # EST timezone
+        # Use the desired time as a seed for the random number generator
+        random.seed(desired_time)
+        # Generate a random 4-digit number
+        random_number = random.randint(1000009, 9999999)
+        # Ensure it's 4 digits long
+        random_number = str(random_number).zfill(random.randint(7, 10))
+        with open(f'book{random_number}.md', 'w') as file:
             file.write(f'# {self.title}\n\n')
             for chapter in self.content:
                 file.write(f'## {self.chapters[self.content.index(chapter)]["title"]}\n\n')
                 for paragraph in chapter:
                     file.write(
-                        f'### {self.chapters[self.content.index(chapter)]["paragraphs"][chapter.index(paragraph)]["title"]}\n\n')
+                        f'### {self.chapters[self.content.index(chapter)]["paragraphs"][chapter.index(paragraph)]["title"]}\n\n'
+                              )
                     file.write(paragraph + '\n\n')
-                file.write('\n\n')
 
     def get_chapter(self, chapter_index, prompt):
-        if len(self.base_prompt) == 3:
+        if len(self.base_prompt) <= 9:
```

Comment:
This increase is because you're using 16k, right? So you can give more context?

Reply:
This might have been the ghostwriter on replit but I'm not sure because I can't remember why I changed it in the first place